src/main/java/com/bookshop/controller/BookController.java

```diff
@@ -34,14 +34,14 @@ public class BookController {
     @GetMapping
     @Operation(summary = "Get all books by page",
             description = "Get a list of all books at x page with y size")
-    @PreAuthorize("hasAnyRole('USER', 'ADMIN')")
+    @PreAuthorize("hasRole('USER')")
     public List<BookDto> getAll(Pageable pageable) {

```

Comment:
u can add annotation @PageableDefault with size, soty and direction parapeters to cofigure Pageable